src/app/shared/nav-tree-data.json

```diff
@@ -0,0 +1,89 @@
+{
```

Comment:
This file is duplicated. Why? 
I think that we should have it in a "common".

Reply:
Addressed on https://github.com/frees-io/freestyle-opscenter-webclient/pull/23/commits/3ea1e64e5dfc35d7c98dbf7f7f9f3ac9391e7607